manifests/plugin/dynflow/params.pp

```diff
@@ -11,8 +11,5 @@
   $ssl_disabled_ciphers  = undef
   $tls_disabled_versions = undef
   $open_file_limit       = 1000000
-  $external_core         = $facts['os']['family'] ? {
-    'RedHat' => true,
-    default  => undef
-  }
+  $external_core         = undef
```

Comment:
Given this type is a boolean, I would think false would work here and be cleaner.

Reply:
It is more of a three valued logic kind of thing, we treat `undef` slightly differently from `false`. I left it at `undef` because that's the way we used to deploy it on debian. Although I must admit that setting it to `false` would be cleaner and probably would work just fine

Reply:
Changed my mind in the end, went with explicit false